proto/api/v2/compliance_results_service.proto

```diff
@@ -41,6 +41,9 @@ message ComplianceCheckResult {
     string                      instructions = 5;
     string                      standard     = 6;
     string                      control      = 7;
+    string                      rationale    = 8;
```

Comment:
Add a `// Next tag:` comment at the top of this message

Reply:
that's a good catch!